components/style/list.css

```diff
@@ -0,0 +1,48 @@
+.list {
+  box-shadow: 0px 0px 1px var(--grey);
+  width: 20rem;
```

Comment:
You can give min or max width for responsiveness

Reply:
Ya, will do that